src/Data/Contract/LocalStorage/localStorageContractDataClass.ts

```diff
@@ -142,13 +142,26 @@ async function attributes(): Promise<DataClassSchema> {
   ] as const
 
   return {
+    agent: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Bearbeiter' : 'Agent',
+        to: 'User',
+      },
+    ],
     cancelationEndAt: [
       'date',
       { title: lang === 'de' ? 'Kündigungsfrist' : 'Cancelation period' },
     ],
     category,
     endAt: ['date', { title: lang === 'de' ? 'Vertragsende' : 'Contract end' }],
-    // TODO: pisa schema: remove internalDepartment for now
+    internalDepartment: [
+      'reference',
+      {
+        title: lang === 'de' ? 'Abteilung' : 'Department',
+        to: 'User',
```

Comment:
* [ ] Q: Just to be sure: The department is represented/modeled by a user?

Reply:
Yes. Internal persons and departments are provided by the user endpoint. As well as the contacts that are in the company structure of the authenticated user.